src/w3c/templates/sotd.js

```diff
@@ -1,6 +1,6 @@
 import hyperHTML from "hyperhtml";
 
-export default conf => {
+export default (conf, opts) => {
```

Comment:
Just wondering, is conf used?

Reply:
Yes, `conf.isMemberSubmission`, `conf.sotdAfterWGinfo`, etc. ESLint warns for us when it's not used 👍

Reply:
ok, makes sense. The thing is, I think `isMemberSubmission` and friends are all generated in headers.js? We could, in theory, start putting those into the `opts` object.

Reply:
(Slight ramble, we don't need to do this now!)

`isMemberSubmission` is used in sotd.js also... previously, I'd thought attaching these things to w3c-defaults.js and making them proper getters, which can be computed.  So, imagine:

```JS
{
   get isMemberSubmission(){
      return this.specStatus === "Member-SUBM" 
   }
}
```